app/src/helpers.js

```diff
@@ -0,0 +1,19 @@
+export function calculateWinner(squares) {
+  const lines = [
+    [0, 1, 2],
+    [3, 4, 5],
+    [6, 7, 8],
+    [0, 3, 6],
+    [1, 4, 7],
+    [2, 5, 8],
+    [0, 4, 8],
+    [2, 4, 6],
+  ];
+  for (let i = 0; i < lines.length; i++) {
```

Comment:
So I think this would be more javasript-y to do this with a Array.reduce rather than a for loop.